### ISSUE TITLE
Theme quirks

### ISSUE THREAD

**Issue by WNYmathGuy** [friendica/friendica#6794]
### Expected behavior
Perfection. :wink: :rofl: 

### Actual behavior
Explaination below.

### Friendica version you encountered the problem
2019.01

### Friendica source (git, zip)
git

### PHP version
7.2.15

### SQL version
MariaDB 10.1.38

### Explaination
I was trying new themes to decide on wich one I liked best and I settled on Frio. I was tuning in my instance when I noticed an odd gost on the Frio search widget. See screenshots below. I'm not sure but I think it may be a misplaced `.com/admin/site/ "Banner/Logo"` field. I couldn't delete it becausee when it's blank the system puts the default back in. If an admin puts in a custom choice for this field, it will be disruptive unless they are really careful about the image size they point to and the number of characters they have in their text. (_seems the Vier theme handles this more wisely than other themes_) Most of the themes throw that content up as preformatted text which is VERY wierd. So Quattro would be my pick as a user favorite but the preformatted HTML in the center of the header is wierd.

The other strange thing about that `.com/admin/site/ "Banner/Logo"` field is that on some themes it points to https://friendi.ca/ and other times it's a home type button for the instances FQDN. This inconsistency seems unuseful. 

I found it better to use the "Footer note" section of the "Impressum" add-on to make an explicit link to the main Friendica site. Top left corners are typically the home button for the site you are already on, so it seems that's the better target to have all themes using it for. 

One thing that went around the Mastodon best practices was to set an instance so that the unauthenticated user who was deciding if they wanted to join that instance would see the public content of the local users. This is a better representation of an instance than the firehose of federated public comments. When I was looking at what a non-user would see for differen't themes, I wasn't too pleased with my favorite choice of the Frio theme. It was beautiful and inviting based on the background image, but their was no way to display the community posts.

### Summary

- Fix the display of the `.com/admin/site/ "Banner/Logo"` field on all themes. 
- Make sure the default page for unauthenticated users shows or has access to the public local community posts.

### Screenshots

**Duepuntozero**
<details>

![duepuntozero screenshot community](https://user-images.githubusercontent.com/28640985/53684447-73cf9880-3cdb-11e9-9521-a2e072633e67.png)

![duepuntozero screenshot login](https://user-images.githubusercontent.com/28640985/53684448-74682f00-3cdb-11e9-9aa5-caf970e05e07.png)

![duepuntozero screenshot search](https://user-images.githubusercontent.com/28640985/53684449-74682f00-3cdb-11e9-902f-2239ff6f42be.png)

![duepuntozero screenshot user](https://user-images.githubusercontent.com/28640985/53684450-74682f00-3cdb-11e9-8063-15f09652b502.png)

</details>
<hr />

**Frio**
<details>

![frio screenshot community](https://user-images.githubusercontent.com/28640985/53684477-e04a9780-3cdb-11e9-8386-fa29ecfa41ba.png)

![frio screenshot login](https://user-images.githubusercontent.com/28640985/53684478-e04a9780-3cdb-11e9-8f1d-d6123eca6b29.png)

![frio screenshot search](https://user-images.githubusercontent.com/28640985/53684479-e0e32e00-3cdb-11e9-8312-db892db83bd3.png)

![frio screenshot user](https://user-images.githubusercontent.com/28640985/53684480-e0e32e00-3cdb-11e9-86d6-f677224f9863.png)

</details>
<hr />

**Quattro**
<details>

![quattro screenshot community](https://user-images.githubusercontent.com/28640985/53684484-eb9dc300-3cdb-11e9-8833-3ce8ec133546.png)

![quattro screenshot login](https://user-images.githubusercontent.com/28640985/53684485-eb9dc300-3cdb-11e9-8346-96a38ccb0409.png)

![quattro screenshot search](https://user-images.githubusercontent.com/28640985/53684486-eb9dc300-3cdb-11e9-8581-7820bc8e8d16.png)

![quattro screenshot user](https://user-images.githubusercontent.com/28640985/53684487-ec365980-3cdb-11e9-93a3-590692c8036e.png)

</details>
<hr />

**Smoothly**
<details>

![smoothly screenshot community](https://user-images.githubusercontent.com/28640985/53684488-f6f0ee80-3cdb-11e9-97eb-bf66df133473.png)

![smoothly screenshot login](https://user-images.githubusercontent.com/28640985/53684489-f6f0ee80-3cdb-11e9-9eca-827b1e7cf2fa.png)

![smoothly screenshot search](https://user-images.githubusercontent.com/28640985/53684490-f6f0ee80-3cdb-11e9-9146-33547c4c6065.png)

![smoothly screenshot user](https://user-images.githubusercontent.com/28640985/53684491-f6f0ee80-3cdb-11e9-9ba0-605f82c5add5.png)

</details>
<hr />

**Vier**
<details>

![vier screenshot community](https://user-images.githubusercontent.com/28640985/53684496-01ab8380-3cdc-11e9-91c2-917380004405.png)

![vier screenshot login](https://user-images.githubusercontent.com/28640985/53684497-01ab8380-3cdc-11e9-8cc3-10bdf65491ad.png)

![vier screenshot search](https://user-images.githubusercontent.com/28640985/53684498-01ab8380-3cdc-11e9-9bce-650dfdf745cb.png)

![vier screenshot user](https://user-images.githubusercontent.com/28640985/53684499-01ab8380-3cdc-11e9-81ad-1c2765db3a1b.png)

</details>

**Comment by MrPetovan** [comment]
Hi and thank you for this incredibly detailed explanations/tries/issues.

In order:
- The `Banner/Logo` display quirks (HMTL code displayed as is) have been stamped out in the upcoming release.
- The Frio theme (at least) disregards the `Banner/Logo` setting completely for the reason you ventured.
- Yes, there could be a link/view of the public local posts on the home page. However we don't have many designers working on Friendica and this may take a while to be actually implemented.

**Comment by poVoq** [comment]
> Make sure the default page for unauthenticated users shows or has access to the public local community posts.

IMHO the main annoyance I have with Frio (besides the incompatibility with the XMPP ConverseJS due to the bootstrap issue).

**Comment by MrPetovan** [comment]
I agree, the home page is a little to terse for my own taste as well.

**Comment by poVoq** [comment]
Maybe by default the Frio homepage could include a version of this:
https://git.friendi.ca/lubuwest/Federation-widget
Similar to how the Mastodon landing page has this "glimpse into the community" thing?

**Comment by MrPetovan** [comment]
Yes, that sounds like a good start.

**Comment by annando** [comment]
We have to make sure that this can be disabled. Because of article 17 I don't want to expose too much content.

**Comment by tobiasd** [comment]
I agree with @annando, if the access to the community page is restricted or the access to the postings of the node, then the landing page should also not show any postings. And not only because of article 17, but in general as some people simply don't like it.
